package.json

```diff
@@ -22,11 +22,12 @@
     "promote": "heroku pipelines:promote -r staging",
     "dump-schema": "babel-node ./scripts/dump-schema.js",
     "heroku-postbuild": "babel index.js config.js -s inline -d build && babel lib -s inline -d build/lib && babel schema -s inline -d build/schema",
-    "pre-commit": "lint-staged"
+    "precommit": "lint-staged"
```

Comment:
As long as this task _only_ runs prettier, then this seems appropriate, because that way there's less indirection when tracing the git history of a file. When doing it from `pre-push` there could be a lot of formatting only changes, which will be the changes you're shown when using e.g. `git blame`.